scripts/test/runtests.sh

```diff
@@ -81,4 +85,7 @@ done
 popd
 set -e
 
+#resetting the path so that we can be good neighbors to other scripts running after this.
+export PATH=$StartPath
```

Comment:
Will it affect any scripts that run after this? Thats's what I was trying to avoid.